package.json

```diff
@@ -167,10 +167,10 @@
     "snazzy": "^9.0.0",
     "split2": "^4.1.0",
     "standard": "^17.0.0-2",
-    "tap": "^16.0.0",
+    "tap": "^16.2.0",
     "tsd": "^0.20.0",
-    "typescript": "^4.5.4",
-    "undici": "^4.11.3",
+    "typescript": "^4.7.2",
+    "undici": "^4.16.0",
```

Comment:
Isn't this at v5.2.0 atm? Or is this PR just doing minor/hotfix bumps?

Reply:
Actually, it is `5.4.0`, and I don't see a big problem on bumping the major of `undici`.
https://github.com/nodejs/undici/releases/tag/v5.0.0

Reply:
bumped undici